README.md

```diff
@@ -8,75 +8,102 @@ We more than welcome contributions to KubeVirt documentation. Please reach out i
 
 ### Get started
 
-- Create fork of GitHub user-guide repo
+1. Create fork of GitHub user-guide repo
 
-- Clone repository, check out source branch and prepare the Jekyll site
-  ```console
-  git clone -b master https://github.com/mygithubname/user-guide.git && cd user-guide
-  ```
+2. Clone repository, check out source branch and prepare the mkdocs site
```

Comment:
We both did bad here...  using source branch was an old idea.  We changed to trusting master and using feature branches.  I forgot to update the README.

The workflow is... 
( clone repo -> cd into repo ) || (checkout master -> create feature branch ) -> do work -> push origin -> create pull_req to master

If you can put that into words I'll approve.

Reply:
Yeah I'll take a look at this, the changes I made here were mostly just switching to numbers instead of dashes, but if that's the workflow laid out I think I might be able to find a decent wording. 

- [ ] Reword the getting started guide #2